flycheck.el

```diff
@@ -7518,9 +7535,8 @@ See URL `https://github.com/eslint/eslint'."
             (flycheck-sanitize-errors errors))
     errors)
   :enabled
-  (lambda (checker)
-    (let* ((default-directory (flycheck-compute-working-directory checker))
-           (executable (flycheck-find-checker-executable checker))
+  (lambda ()
+    (let* ((executable (flycheck-find-checker-executable 'javascript-eslint))
```

Comment:
Did you really mean to remove the default-directory here?  Will the call to `executable` return the same thing?